json-schemas/interface-methods/records-query.json

```diff
@@ -38,6 +38,19 @@
         "filter": {
           "$ref": "https://identity.foundation/dwn/json-schemas/records-filter.json"
         },
+        "pagination": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "limit": {
+              "type": "number",
+              "minimum": 0
```

Comment:
Sanity: what does it mean to have limit being zero? Should the minimum be 1?

Reply:
This is an oversight, it should be 1.